src/Grpc.Net.Client/Configuration/LoadBalancingConfig.cs

```diff
@@ -0,0 +1,54 @@
+﻿#region Copyright notice and license
+
+// Copyright 2019 The gRPC Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#endregion
+
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Grpc.Net.Client.Configuration
+{
+    /// <summary>
+    /// Base type for load balancer policy configuration.
+    /// </summary>
+    public class LoadBalancingConfig : ConfigObject
+    {
+        /// <summary>
+        /// <c>pick_first</c> policy name.
+        /// </summary>
+        public const string PickFirstPolicyName = "pick_first";
```

Comment:
Why snake_case? I think I saw some docs in grpc proposals with all caps: https://github.com/grpc/proposal/blob/c5722a35e71f83f07535c6c7c890cf0c58ec90c0/A42-xds-ring-hash-lb-policy.md#xds-api-fields-1. Also, unless there's compatibility reasons for snake casing, should we use camelCase instead like the other string constants like https://github.com/grpc/grpc-dotnet/blob/9f72e70c6a47ae823bb3ddcc719f1c63bf721fc7/src/Grpc.Net.Client/Configuration/RetryPolicy.cs#L31? Granted, these string should use whatever case that makes the most sense for the component so if other languages' client load balancing implementations use snake_case, that makes sense too.

Reply:
Matches values from https://github.com/grpc/grpc-proto/blob/ec886024c2f7b7f597ba89d5b7d60c3f94627b17/grpc/service_config/service_config.proto#L328-L373